tests/test_display.py

```diff
@@ -81,6 +96,23 @@ def test_display_default(self, mock_put_text):
                                          position=(224, 294),
                                          thickness=self.thickness)
 
+    @patch('sense.display.DisplayClassnameOverlay._display_class_name')
+    def test_display_default_within_duration(self, mock_display_class_name):
+        test_display = base_display.DisplayClassnameOverlay(thresholds={'Dabbing': 0.5}, duration=.1)
+        start_time = time.perf_counter()
+        test_display.display(self.img, {'sorted_predictions': [['Dabbing', 0.6]]})
+
+        call_count = 0
+        while time.perf_counter() - start_time <= test_display.duration:
+            test_display.display(self.img, {'sorted_predictions': [['Dabbing', 0.6]]})
+            mock_display_class_name.assert_called_with(self.img, 'Dabbing')
+            call_count = mock_display_class_name.call_count
+
+        # Display something that's not in threshold
```

Comment:
This comment is confusing. Shouldn't it rather be "Display nothing after duration has passed"?